portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,89 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Samiksha Kapoor</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+
+<body>
+  <div class='Content'>
+    <div class='Display'>
+      <img class='ProfilePic' src='images/my_photo.jpg'>
```

Comment:
It's advised to add 'alt' property to img tag. This is to provide the description of image in case image can't be rendered due to connection issues.